.readthedocs.yml

```diff
@@ -7,10 +7,6 @@ build:
   jobs:
     pre_build:
       - cp -r examples docs/source/
-    post_install:
-      - pip install --no-cache-dir poetry
-      - poetry export -f requirements.txt -o requirements.txt -E all --without-hashes
```

Comment:
This change is because `poetry export` fails while it works perfectly with lock and install.

```
➜  RecTools git:(feature/py313) poetry export -E all --without-hashes
Warning: poetry-plugin-export will not be installed by default in a future version of Poetry.
In order to avoid a breaking change and make your automation forward-compatible, please install poetry-plugin-export explicitly. See https://python-poetry.org/docs/plugins/#using-plugins for details on how to install a plugin.
To disable this warning run 'poetry config warnings.export false'.

Dependency walk failed at numpy (>=1.17.0)
```

It may relate to https://github.com/python-poetry/poetry-plugin-export/issues/176